### PR TITLE
Matter Linux Robotic Vacuum Cleaner Example README.md

### PR DESCRIPTION
**Pull request by lboue**
Add README.md for [Matter Linux Robotic Vacuum Cleaner Example](https://github.com/project-chip/connectedhomeip/tree/master/examples/rvc-app/linux)

### CODE REVIEW

**Comment by lboue** [comment]
I don't understand the root cause of this error or how to correct the problem:

```
copying images... [100%] testing/img/unit_tests.png

copying static files... done
copying extra files... done
dumping search index in English (code: en)... done
dumping object inventory... done
build finished with problems, 1 warning.
make: *** [Makefile:21: html] Error 1
Error: Process completed with exit code 2.
```

File: examples/rvc-app/linux/README.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+    Use specific bluetooth interface for BLE advertisement and connections.
+
+    `interface id`: the number after `hci` when listing BLE interfaces by
+    `hciconfig` command, for example, `--ble-device 1` means using `hci1`
+    interface. Default: `0`.
+
+## Running the Complete Example on Raspberry Pi 4
+
+> If you want to test Echo protocol, please enable Echo handler
```
Comment (andy31415) [inline examples/rvc-app/linux/README.md:71]: why is the echo protocol referenced here?

I see this in a lot of README.md inside linux ... but generally it seems out of place.

Comment (lboue) [reply examples/rvc-app/linux/README.md]: This is a copy/paste from another example. I've removed this section.

**Comment by lboue** [comment]
I don't understand the root cause of this error or how to correct this:

```
copying images... [100%] testing/img/unit_tests.png

copying static files... done
copying extra files... done
dumping search index in English (code: en)... done
dumping object inventory... done
build finished with problems, 1 warning.
make: *** [Makefile:21: html] Error 1
Error: Process completed with exit code 2.
```

**Comment by andy31415** [comment]
> I don't understand the root cause of this error or how to correct this:
> 
> ```
> copying images... [100%] testing/img/unit_tests.png
> 
> copying static files... done
> copying extra files... done
> dumping search index in English (code: en)... done
> dumping object inventory... done
> build finished with problems, 1 warning.
> make: *** [Makefile:21: html] Error 1
> Error: Process completed with exit code 2.
> ```

@lboue it seems the error is way earlier:

```
pickling environment... done
/home/runner/work/connectedhomeip/connectedhomeip/matter/docs/_build/src/examples/rvc-app/linux/README.md: WARNING: document isn't included in any toctree
checking consistency... done
```

I think you need to mark it as orphaned or add it as part of a TOC tree.

**Comment by lboue** [comment]
> I think you need to mark it as orphaned or add it as part of a TOC tree.

@andy31415 

It seems that `orphan` is not allowed
```
/home/runner/work/connectedhomeip/connectedhomeip/matter/docs/_build/src/examples/rvc-app/linux/README.md:137: WARNING: 'toctree': Unknown option keys: ['orphan'] (allowed: ['caption', 'glob', 'hidden', 'includehidden', 'maxdepth', 'name', 'numbered', 'reversed', 'titlesonly']) [myst.directive_parse]
looking for now-outdated files... none found
```

**Review by andy31415 — APPROVED** [review]

**Comment by andy31415** [comment]
@lboue could you fix the merge conflicts here? I believe the toc bits were moved around recently so resolving the conflict is not easy. See  #36100

**Comment by andy31415** [comment]
Will likely also need restyle and documentation build fixes. Will fast track it once everything is green.

**Pull request closed without merging**